app/scripts/directives/processTemplateDialog.js

```diff
@@ -43,6 +68,34 @@
 
     ctrl.$onInit = function() {
       ctrl.loginBaseUrl = DataService.openshiftAPIBaseUrl();
+      ctrl.preSelectedProject = ctrl.selectedProject = ctrl.project;
+      listProjects();
+
+      ctrl.projectEmptyState = {
+        icon: 'pficon pficon-info',
+        title: 'No Project Selected',
+        info: 'Please select a project from the dropdown to load Templates and Images from that project.'
```

Comment:
'Please select a project from the dropdown to load templates from that project.'